cortipy/cortical_client.py

```diff
@@ -73,13 +73,7 @@ def __init__(self,
     else:
       self.apiKey = os.environ["CORTICAL_API_KEY"]
     self.apiUrl = baseUrl
-    
-    # Create cache directory if necessary
-    cacheDir = os.path.join(cacheDir, retina)
-    if not os.path.exists(cacheDir):
-      os.makedirs(cacheDir)
```

Comment:
@BoltzmannBrain This was probably copied from the `pycept` constructor, but it wrong in hindsight. The constructor of an object has a [single responsibility](http://en.wikipedia.org/wiki/Single_responsibility_principle), which is to construct the object. Any other actions being taken in a constructor are misplaced, including making API calls, system calls, file system operations, etc.

The test I added asserts against file system operations, so this has been removed and achieved lazily in the function that writes to the cache.